website/docs/user-group-role/roles/manage_roles.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Manage roles"
+description: "Learn how to work with roles and permissions in authentik."
+---
+
+Roles are a collection of permissions, which can then be assigned, en masse, to a group. Using roles is a way to quickly grant permissions; by adding a user to the group with the appropriate assigned roles, any user in that group then inherits all of those permissions that are assigned to the role.
```

Comment:
Semicolons are the work of the devil.  Consider a colon or maybe even just two independent sentences.